doc/Changelog.md

```diff
@@ -3,6 +3,18 @@ Changelog {#changelog}
 
 # git master {#master}
 
+* [#10](https://github.com/BlueBrain/Fivox/pull/10)
+  The voxelize app now generates clipped volumes, limiting its extent to the
+  relevant data, being now consistent with the data source for Livre.
+  Move the calculation of the cutoff distance (still based on the maxError) to
+  the different loaders.
+* [#10](https://github.com/BlueBrain/Fivox/pull/10)
+  Rescale to output type range: when writing into disk (voxelize tool) and the
+  output data type is different than float, rescale according to output data
+  type. The same rescale operation is applied in the Livre data source, to
+  rescale from float to uint8_t.
+  All the rescale/cast operations are done at the end of the workflow, not in
+  the loaders/functors. Tests adapted accordingly.
```

Comment:
This is a changelog. Lines should be one, max two lines with more text in the PR or commit if needed, e.g.:
* voxelize: Limit volume to area covered by events.